src/main/java/com/dataloom/edm/EdmApi.java

```diff
@@ -116,25 +117,25 @@ Schema getSchemaContents(
      * @param name
      * @param entityTypes
      */
-    @PUT( SCHEMA_BASE_PATH + NAMESPACE_PATH + NAME_PATH )
+    @PUT( SCHEMA_BASE_PATH + "/" + NAMESPACE_PATH + "/" + NAME_PATH )
     Void addEntityTypesToSchema(
             @Path( NAMESPACE ) String namespace,
             @Path( NAME ) String name,
             @Body Set<FullQualifiedName> entityTypes );
 
-    @DELETE( SCHEMA_BASE_PATH + NAMESPACE_PATH + NAME_PATH )
+    @HTTP(method="DELETE", path = SCHEMA_BASE_PATH + "/" + NAMESPACE_PATH + "/" + NAME_PATH, hasBody = true )
```

Comment:
This is a workaround for delete with body in retrofit:
http://stackoverflow.com/questions/37942474/delete-method-is-not-supportingnon-body-http-method-cannot-contain-body-or-t